Dysplasia.html

```diff
@@ -1512,6 +1481,26 @@ <h2 class="text-center">Dysplasia Form v 0.01</h2>
         toggleValidationMessages(validateAlone($(this), colposcopyresultsValues, colposcopyresultsMessage));
     });
 
+    /* Start - Checking that the user entered provider exists in the list of possible providers */
```

Comment:
Should this go to the core muzima javascript as the implementation seems to be similar in all instances?